db/migrate/20240813134450_create_setting_for_github_email.rb

```diff
@@ -0,0 +1,9 @@
+class CreateSettingForGithubEmail < ActiveRecord::Migration[7.1]
+  def change
+    LumenSetting.create!(
+      name: 'Email to search for to find the exsting GitHub user',
+      key: 'github_user_email',
+      value: 'gh@email.com'
```

Comment:
What should this email actually be?

Reply:
We will configure it before we run in for the first time.